autorest-core/dist/language-service/document-analysis.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
if you want to trim things, the language-serverice folder is probably something you can delete